targets/targets.json

```diff
@@ -15,6 +15,7 @@
         "default_lib": "std",
         "bootloader_supported": false,
         "static_memory_defines": true,
+        "components_add": ["FPGA_CI_TEST_SHIELD"],
```

Comment:
```suggestion
```
I think we should move this to `"components": ["PSA_SRV_IMPL", "PSA_SRV_EMUL", "NSPE", "FPGA_CI_TEST_SHIELD"],` above.

Reply:
Nice catch! Probably we need new PR.